client/src/index.css

```diff
@@ -2,11 +2,13 @@
 @tailwind components;
 @tailwind utilities;
 
-.group:hover .group-hover\:flex {
-  display: flex;
+@layer base {
+  body {
+    @apply bg-teal-light;
+  }
 }
 
 .modal {
   width: clamp(10%, 500px, 50%);
   min-height: 400px;
```

Comment:
Somewhere in our merge conflict resolutions, the line `max-height: 100%;` got lost, and with it our modals lost the ability to have scroll bars. With your blessing, I am going to add it again.